src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Nrbf/SerializationRecordExtensions.cs

```diff
@@ -0,0 +1,357 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+
+using System.Collections;
+using System.Drawing;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+
+#nullable enable
+namespace System.Formats.Nrbf
```

Comment:
I've used this namespace because there is plenty of types in `System.Windows` like `ClassRecord` or `ArrayRecord` that conflict with types with exact same names from the `System.Formats.Nrbf` namespace.